tests/foreman/ui/test_discoveredhost.py

```diff
@@ -143,6 +145,183 @@ def test_positive_pxe_less_discovery(self):
                 self._assertdiscoveredhost(hostname)
                 self.assertIsNotNone(self.discoveredhosts.search(hostname))
 
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_with_dhcp_semiauto(self):
```

Comment:
Yeah.. thanks for pointing this. I was thinking while adding test stubs. Few test stubs are difficult to automate like: validating all buttons on discovery status screen. Will see if I can find anyway to automate.